core/kazoo_data/src/kzs_attachments.erl

```diff
@@ -174,22 +173,23 @@ attachment_handler_jobj(Handler, Props) ->
     JObj = kz_json:from_list(props:get_value('attachment', Props, [])),
     kz_json:set_value(kz_term:to_binary(Handler), JObj, kz_json:new()).
 
--spec handle_put_attachment(att_map(), kz_json:object(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary()
+-spec handle_put_attachment(att_map(), kz_term:api_object(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary()
                            , kz_term:proplist(), kz_term:proplist()) ->
                                    {'ok', kz_json:object()} |
                                    {'ok', kz_json:object(), kz_term:proplist()} |
                                    data_error().
 
-handle_put_attachment(#{att_post_handler := 'stub'
```

Comment:
@danielfinke @jamesaimonetti why was this removed ? `att_post_handler := 'stub'`

Reply:
nm